src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
```

Comment:
This is from me setting a bad example. Rachel can you fix it here and I'll fix it in the other files (bowl, plate, etc)?